react-js/react.md

```diff
@@ -4,26 +4,29 @@
 
 8 Hours
 
-### Prerequisites
+- Lesson : 1 Hr
+- Examples: 30-45 mins
+- Guided Practice: 2 Hrs
```

Comment:
Are these still accurate?

Reply:
These are approximate time to spend for each part.

Reply:
Please suggest changes if any required.